dot/state/block.go

```diff
@@ -250,6 +251,37 @@ func (bs *BlockState) GetHashByNumber(num uint) (common.Hash, error) {
 	return common.NewHash(bh), nil
 }
 
+// GetHashesByNumber returns the block hash on our best chain with the given number
+func (bs *BlockState) GetHashesByNumber(num uint) ([]common.Hash, error) {
```

Comment:
```suggestion
func (bs *BlockState) GetBestHashesByNumber(num uint) ([]common.Hash, error) {
```
Let's make it more explicit that it's traversing the best chain that we have.

Reply:
my bad! The comment was not accurate. I fixed that.

Best chain means best fork. So, best chain hashes for given number would not make sense, since there would be only one.

here, GetBlockByNumber gets the block from best chain, but later we can use that block to get all the blocks on that level.